p/scripts/main.js

```diff
@@ -1286,7 +1277,54 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var $sidebar = null;
+var useJsScrollbar = !(parseInt(getComputedStyle(document.getElementById('sidebar'), '::-webkit-scrollbar').width) < 99);
```

Comment:
A few notes:
* `parseInt()` will return `NaN` when the property is empty (e.g. in Firefox), and `NaN < 99 == false`;
* Edge / IE would return the width of the whole sidebar, which is about ~300.